submissions/Westerinoo/HTML-CSS-popup/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Pop-Up</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+    <nav class="navigation">
+        <ul class="nav__lists">
+           <li class="nav__items"><a class="link" href="#">Mail</a></li>
+           <li class="nav__items"><a class="link" href="#">Images</a></li>
+           <li class="nav__items">
+           
+                <input id="checkbox__popup" type="checkbox" class="checkbox__popup">
+                <div class="wrap-popup">
+                <div class="popup__content"> 
+                        <ul class="popup__lists">
+                           <li class="popup__items"><a class   ="link-popup" href="#"><img class="icons" src="./icons/calendar.png" alt="icon-calendar"><span>calendar</span></a></li>   
```

Comment:
For decorative images implemented with `img` tag `alt` attribute must be an empty value. Icons are decorative images. Please set an empty value to `alt` attribute.